dbt_project.yml

```diff
@@ -13,10 +13,19 @@ clean-targets:         # directories to be removed by `dbt clean`
   - "target"
   - "dbt_packages"
 
-#vars:
-#    start_date: "20210120" # Defines the earliest GA4 _TABLE_SUFFIX to load into base events model. 
-#    project: "bigquery-public-data"
-#    dataset: "ga4_obfuscated_sample_ecommerce"
+profile: obfuscated_test_db
```

Comment:
@dgitis I don't think we need/want to define a profile or vars in the package itself. They should be defined in the project that loads the package.

Reply:
I wasn't using this. It's been in there from the beginning.